### DISCUSSION TITLE
Carbon future

### DISCUSSION THREAD

**Discussion by reinisg** [carbon-design-system/carbon#15170] [Help] [answered]
Hello, I`m interested in how are plans for future. Does Carbon will switch all components to Web Components or try maintain (Angular, Vue, React and Web components)? 
Maybe there are other features what Carbon will provide?

**Comment by prx25** [comment]
Hello, by the way, I have a question about when the v11 upgrade for Carbon Web Components is expected.

**Comment by tay1orjones** [comment, marked as answer]
Wow this is a year old! Sorry we missed this. 

The core Carbon team will be maintaining two flagship offerings for the foreseeable future, `@carbon/react` and `@carbon/web-components`. We're aiming for complete feature parity between the two and are working to ensure updates are made to both packages as close to simultaneously as possible. The v2 of `@carbon/web-components` is out now and is compatible with v11.

Community-maintained offerings for other frameworks will continue to exist, though they aren't maintained by the core team and may lag slightly behind in features or styling. Hope this helps!

**Reply by reinisg** [reply]
Great!! Thank you for your response!
